eng/build.yml

```diff
@@ -26,6 +26,10 @@ parameters:
   # Depends on 
   dependsOn: ''
 
+  runtimeFeedParams:
```

Comment:
Is this necessary? I don't know yaml very well, but they don't seem to be used in build.yml.

Reply:
Oops, taking this away. Was trying not to use the feature as it's really new. But all other options were not elegant and made a lot of assumptions. Cleaning this up now.